dotcom-rendering/src/web/components/PinnedPost.tsx

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import {
+	neutral,
+	space,
+	textSans,
+	news,
+	from,
+	visuallyHidden,
+	transitions,
+	focusHalo,
+	height,
+} from '@guardian/source-foundations';
+import {
+	SvgMinus,
+	SvgPlus,
+	SvgPinned,
+} from '@guardian/source-react-components';
+import { timeAgo } from '@guardian/libs';
+import { css } from '@emotion/react';
+
+const pinnedPostContainer = css`
+	border: 3px solid ${news[300]};
+	padding-bottom: ${space[1]}px;
+	margin-bottom: ${space[9]}px;
+	position: relative;
+	background: ${neutral[100]};
+	#pinned-post-checkbox:checked ~ #collapsible-body {
+		max-height: fit-content;
+	}
+	#pinned-post-checkbox:checked ~ #overlay,
+	#pinned-post-checkbox ~ label #svgminus,
+	#pinned-post-checkbox:checked ~ label #svgplus {
+		display: none;
+	}
+	#pinned-post-checkbox ~ label #svgplus,
+	#pinned-post-checkbox:checked ~ label #svgminus {
+		display: block;
+	}
+	#pinned-post-checkbox ~ label::after {
+		content: 'Show more';
+	}
+	#pinned-post-checkbox:checked ~ label::after {
+		content: 'Show less';
+	}
+`;
+
+const pinnedPostRow = css`
+	background: ${news[300]};
+	height: 2rem;
```

Comment:
Could we change these rems to pixels?

I realise it's somewhat arbitrary and that there are pros and cons to both approaches but in DCR the convention is pixels so it's really just for consistency.